pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = [ "setuptools", "setuptools_scm" ]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ts_ess_labjack"
+description = "CSC to make the dome for the Rubin Observatory Simonyi Survey Telescope dome track the telescope."
+license = { text = "GPL" }
+classifiers = [ "Programming Language :: Python :: 3" ]
+urls = { documentation = "https://ts-ess-labjack.lsst.io", repository = "https://github.com/lsst-ts/ts_ess_labjack" }
+dynamic = [ "version" ]
+
+[tool.setuptools.dynamic]
+version = { attr = "setuptools_scm.get_version" }
+
+[tool.setuptools.packages.find]
+where = [ "python" ]
+
+[project.scripts]
```

Comment:
Is this section needed at all? Prior to introducing `pyproject.toml` this project didn't have any scripts at all. We don't intend to run this as a stand alone application but only as a "plugin" for the ESS CSC, right?